### ISSUE TITLE
some loss keep increase

### ISSUE THREAD

**Issue by imyzx2017** [hanzhanggit/StackGAN-Pytorch#24]
What's the property parameters for training CUB-birds(200-2011) datasets?

I had tried several param_sets, but the G loss always increases and oscillates in 2.0 after 50 epochs, and the KL loss is always increase..

How to improve the trend of loss to get better images? Any suggestions? Thanks!

**Comment by Aarthif-Nawaz** [comment]
Hi @imyzx2017 , same for me as well, when i train for more epochs the loss of the Generator keeps increasing. and its at 5.0 after 350 epochs. Any quick fix you got.
